rpm-ostree/containerfiles/unstable

```diff
@@ -53,7 +53,8 @@ RUN ${CMD_INSTALL} \
   && ostree container commit
 
 RUN mkdir -p /run/akmods; for k in $(ls -1 /usr/src/kernels); do \
-  akmods --force --kernels "${k}" --kmod "nvidia" || exit 1; done; rm -r -f /run/akmods \
+  akmods --force --kernels "${k}" --kmod "nvidia" || exit 1; \
+  ls /usr/lib/modules/"${k}"/extra/nvidia/nvidia.ko.xz || exit 1; done; rm -r -f /run/akmods \
```

Comment:
I am a little concerned that it may be possible for the akmods command to fail, but still generate a bad/broken `nvidia.ko.xz` file.